src/pages/index.js

```diff
@@ -4,114 +4,283 @@ import FormValidator from "../components/FormValidator.js";
 import PopupWithForm from "../components/PopupWithForm.js";
 import PopupWithImage from "../components/PopupWithImage.js";
 import Section from "../components/Section.js";
-import {
-  initialCards,
-  formValidationSettings,
-  profileEditButton,
-  profileTitleInput,
-  profileDescriptionInput,
-  addCardForm,
-  addNewCardButton,
-  profileEditForm,
-  cardListEl,
-} from "../utils/constants.js";
 import UserInfo from "../components/UserInfo.js";
+import PopupConfirm from "../components/PopupConfirm.js";
+import Api from "../components/Api.js";
 
-/* Elements */
+/* -------------------------------------------------------------------------- */
+/*                                  Variables                                 */
+/* -------------------------------------------------------------------------- */
 
-//an instance of PopupWithForm
-const profileEditPopup = new PopupWithForm(
-  "#profile-edit-modal",
-  handleProfileEditSubmit
-);
+const config = {
```

Comment:
From the 8th sprint all constants (including `initialCards` and `config`) should be in `constants.js` in the `utils` folder to clean up the main file.